src/ipc/StoreMap.h

```diff
@@ -113,6 +115,8 @@ class StoreMapAnchor
     /// where the updated chain prefix containing metadata/headers ends [update]
     /// if unset, this anchor points to a chain that was never updated
     std::atomic<StoreMapSliceId> splicingPoint;
+
+    RandomUuid *varyUuid;
```

Comment:
This raw pointer looks out of place in a class which instances are normally stored in shared memory.